requirements.txt

```diff
@@ -1,6 +1,7 @@
-DLite-Python>=0.3.3,<1
+DLite-Python>=0.3.3,<0.4
```

Comment:
Why not latest dlite-python?

Reply:
Because it leads to segfault in codecov so that the CI/CD won't run on GitHub.

I haven't had time to dig into exactly why it fails, but there is an old bug that leads to segfault when iterating over storages. I tried to look at it before releasing v0.4.0, but was not able to solve the issue. My guess is that the changes I made, made the bug slightly more exposed such codecov is now able to find and run the problematic code and thereby segfaults. 

It is definitely a bug that should be fixed. But iterating over storages is not something we have documented or is a critical feature for using dlite. Still, I think that it will be very useful to be able to easily list all available storages-